assets/src/edit-story/components/dropDown/index.js

```diff
@@ -194,7 +201,8 @@ function DropDown({ options, value, onChange, disabled, ariaLabel }) {
     (offset) => {
       const findIndex = options.findIndex(
         (item) =>
-          focusedValue && item.value.toString() === focusedValue.toString()
+          !isNullOrUndefined(focusedValue) &&
```

Comment:
Looks like the same thing ( `!isNullOrUndefined` with `item.value.toString() === focusedValue.toString()` ) is repeated 3 times, should perhaps there be a helper for that instead?

Reply:
Yes, used foundIndex useMemo replacement